cloud/amazon/ec2_ami.py

```diff
@@ -44,7 +44,7 @@
     default: 300
   state:
     description:
-      - create or deregister/delete image
+      - create (present), register (register), deregister/delete image (absent)
```

Comment:
How about specifying instance_id does create and specifying image_location does the registration instead?